### ISSUE TITLE
Can't view images linked from Google Drive

### ISSUE THREAD

**Issue by mileaminute** [ProjectNami/projectnami#212]
I link to images located on Google Drive in my posts.

With standard WP I could see images on ios devices, but with Project Nami WP I usually can't see them (they're just bordered squares) - but sometimes I can. I can access them directly on ios from the same link. Also I can see the images with no problems on multiple browsers on a Windows PC.

Has anyone seen this issue or can you offer any troubleshooting suggestions? thanks.

**Comment by patrickebates** [comment]
Is there a plugin you are using for this, or is it simply a direct image reference in the body?
Is your Google Drive doing any checking of the referrer to verify the site trying to use the image?
Got an example?

**Comment by mileaminute** [comment]
It was just a direct image reference. It seems to work in browsers except on IOS. It also seems to be intermittent (when I made a change to the source url to another file service then that image and the other images would show up in Safari on IOS).

Anyway, before having you spend time on this I wanted to test against a non-Project Nami install. Finally had a chance to do that and found the same result so it's not related to Nami. Thanks for your quick response and offer to help. Hopefully this will help others (I ended up using OneDrive, a much better image embedding solution than Google Drive).
